homeassistant/components/zwave/__init__.py

```diff
@@ -830,6 +830,25 @@ def __init__(self, values, domain):
 
     def network_value_changed(self, value):
         """Handle a value change on the network."""
+        if (value.command_class == const.COMMAND_CLASS_CENTRAL_SCENE and
+                value.node.node_id == self.node.node_id and
+                self.hass is not None):
+
+            scene_id = int(str(value.index) + str(value.data))
+
+            _LOGGER.info(
```

Comment:
We can skip the log output if there's nothing else to add. Hass core will already log the event.